src/pages/apps/[id]/deployments/[deployment-id]/Deployment.spec.js

```diff
@@ -15,7 +15,7 @@ describe(fileName, () => {
   describe("when the deployment is successful", () => {
     beforeEach(() => {
       deploy = data.mockDeploymentResponse().deploy;
-      scope = nocks.mockFetchDeploymentsCall({ app, deploy });
+      scope = nocks.mockFetchDeploymentCall({ app, deploy });
```

Comment:
Renamed this function because it was wrongly named. It returns a single deployment, therefore should be named `mockFetchDeploymentCall`.